packages/mux-player/src/index.ts

```diff
@@ -621,6 +627,10 @@ class MuxPlayerElement extends VideoApiElement {
     this.setAttribute(PlayerAttributes.AUDIO, '');
   }
 
+  get hotKeys() {
+    return this.#hotKeys;
+  }
+
   get nohotkeys() {
```

Comment:
hmm I feel like we're not following the convention here. should be 

```suggestion
  get noHotKeys() {
```

maybe in the props map access `this.hasAttribute(PlayerAttributes.NOHOTKEYS)` directly

Reply:
if the attribute name is `nohotkeys`, the property name should also be `nohotkeys` (based on our translation, which assumes hyphens in attributes denote camel case capitalization)

Reply:
does that mean I should rename `get hotKeys` to `get hotkeys`?

Reply:
And should it be renamed in media-chrome too? (I haven't merged that PR yet)